app/views/account/passwd.scala

```diff
@@ -12,7 +12,13 @@ object passwd {
   def apply(form: play.api.data.Form[_])(implicit ctx: Context) =
     account.layout(
       title = trans.changePassword.txt(),
-      active = "password"
+      active = "password",
+      evenMoreJs = frag(
+        embedJsUnsafeLoadThen("""
+          lichess.loadModule('passwordComplexity').then(() =>
+            passwordComplexity.addPasswordChangeListener('form3-newPasswd1')
```

Comment:
Should I factor `form3-newPasswd1` out into a constant (here and in /signup)? Or is it unlikely that the password input id will ever change?

Reply:
meh that doesn't matter either way